packages/website/src/content/docs/docs/actions/car-setup/setup-engine.md

```
@@ -3,25 +3,78 @@ title: Setup Engine
-| Boost Level Decrease | Decrease boost level |
-| Launch RPM Increase | Increase launch control RPM |
-| Launch RPM Decrease | Decrease launch control RPM |
+Select the mode from the **Setting** dropdown in the Property Inspector. Every engine mode is directional — pick **Increase** or **Decrease** in the **Direction** setting to control what a button press does.
 
-## Encoder Support
+### Engine Power
 
-Yes — rotate the dial to increase or decrease the selected engine setting.
+Adjust the engine power setting.
+
+##### Details
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)